content/tutorials/python-whirlwind-tour.md

```diff
@@ -520,11 +520,11 @@ else:
 
 Run the code and check your prediction.
 
-### Age Discrimination Tool
+### Age discrimination tool
```

Comment:
Maybe we shouldn't call it this? How about "Age detection tool" or similar?

Reply:
907c136